website/client/components/avatar.vue

```diff
@@ -1,6 +1,6 @@
 <template lang="pug">
 .avatar(:style="{width, height, paddingTop}", :class="backgroundClass", @click.prevent='castEnd()')
-  .character-sprites
+  .character-sprites(:style='{margin}')
```

Comment:
i think this margin prop is confusing as it's passed to avatar but applies to an element inside